roles/lib_openshift/src/test/unit/test_oc_serviceaccount_secret.py

```diff
@@ -30,6 +30,7 @@
     import ruamel.yaml as yaml  # noqa: EF401
     YAML_TYPE = 'ruamel'
 except ImportError:
+    import yaml
     YAML_TYPE = 'pyyaml'
```

Comment:
@detiber I think we should change this test to exercise both YAML libs, if we support both.
I'd be fine to open and issue and do this in a separate PR.